docs/contributing.md

```diff
@@ -1,42 +1,46 @@
 # Contributing
 
-You are very welcome to contribute in any way. This document is the guide of contribution. it has no intention to limit, only make it efficient.
+We welcome any form of contribution.
 
-- [Provide feedback or ask a question](#provide-feedback-or-ask-a-question)
-- [File a bug](#file-a-bug)
-- [Contribute more](#contribute-more)
-  - [Contribute fixing or extension](#contribute-fixing-or-extension)
-  - [Contribute feature](#contribute-feature)
-- [Code of Conduct](#code-of-conduct)
+Please read and follow the guides below to help not only us but also you to get more efficient.
 
-## Provide feedback or ask a question
+- [Contributing](#contributing)
+  - [Provide feedback or ask a question](#provide-feedback-or-ask-a-question)
+  - [Propose a new feature](#propose-a-new-feature)
+  - [Report a bug](#report-a-bug)
+  - [Contributing fixes or extensions](#contributing-fixes-or-extensions)
+  - [Code of Conduct](#code-of-conduct)
+- [Thank you!](#thank-you)
 
-You are welcome to provide feedback by [filing an issue](https://github.com/microsoft/lisa/issues/new).
+Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and do, grant us the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
 
-When there is any question, please check [FAQ and troubleshooting](troubleshooting.md) first. If there is no satisfying answer, don't hesitate to [file an issue](https://github.com/microsoft/lisa/issues/new).
+When you submit a pull request (PR), a CLA bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
 
-## File a bug
+## Provide feedback or ask a question
 
-Before file a bug, please try to search keywords on existing bugs. If you cannot find any similar bugs, feel free to [file a new one](https://github.com/microsoft/lisa/issues/new).
+You are welcome to provide any feedback by [filing an issue](https://github.com/microsoft/lisa/issues/new).
 
-## Contribute more
+When there is any question, please check [FAQ and troubleshooting](troubleshooting.md) first. If there is no satisfying answer, don't hesitate to [ask us](https://github.com/microsoft/lisa/issues/new).
 
-This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+## Propose a new feature
```

Comment:
There are two level of contribution on features. 1. propose only, 2. propose and implement.
We encourage users on both options.